src/components/AddItem.js

```diff
@@ -1,10 +1,35 @@
 import React, { useState } from 'react';
 import { writeToFirestore, timestamp } from '../lib/firebase';
+import {
+  FormControl,
+  FormLabel,
+  RadioGroup,
+  FormControlLabel,
+  Radio,
+  Typography,
+  Divider,
+  Box,
+} from '@material-ui/core';
```

Comment:
Sometimes teams like to alphabetize these imports...something to think and talk about with the team. In my experience it is difficult to maintain unless there is some kind of linting rule in place.